mk/uvmt/uvmt.mk

```diff
@@ -320,8 +320,8 @@ embench: $(EMBENCH_PKG)
 #   Compare the log against the tracer log.
 #   This checks that sampling went correctly without false positives/negatives.
 
-ISACOV_LOGDIR = $($(SIMULATOR_UC)_RESULTS)/$(CFG)/$(TEST)_$(RUN_INDEX)
-ISACOV_TRACELOG = $(ISACOV_LOGDIR)/trace_core_00000000.log
+ISACOV_LOGDIR = $(SIM_CFG_RESULTS)/$(TEST)/$(RUN_INDEX)
+ISACOV_TRACELOG = $(ISACOV_LOGDIR)/uvm_test_top.env.rvfi_agent.trn.log
 ISACOV_AGENTLOG = $(ISACOV_LOGDIR)/uvm_test_top.env.isacov_agent.trn.log
 
 isacov_logdiff:
```

Comment:
I updated the isacov_logdiff script, now that the old "tracer" has been replaced with the rvfi log.
This has been really useful in checking that coverage is done right, as it uncovered many mistakes. (Using "illegal_bins" also helped in this regard.)
Idea: Could run a "full" regression, but substitute the `test` make-target with `test isacov_logdiff` to check for each test that coverage was sampled correctly. (One blocker is that HINT instructions are not shown with objdump info in the rvfi log.)

Reply:
If feasible, I think it would be nice to have the run-index in the filename of the log itself, in case log files are moved around, it would still be obvious to which run they belong.

Reply:
While that might be the case, I don't think I'll dare touching that infrastructure as part of this PR.